client/src/extension.ts

```diff
@@ -7,13 +7,13 @@ export function activate(context: ExtensionContext) {
   // The server is implemented in node
   let serverModule = context.asAbsolutePath(path.join('server', 'server.js'));
   // The debug options for the server
-  let debugOptions = { execArgv: ["--nolazy", "--debug=6004"] };
+  let debugOptions = { execArgv: ["--nolazy", "--debug=6009"] };
 
   // If the extension is launched in debug mode then the debug server options are used
   // Otherwise the run options are used
   let serverOptions: ServerOptions = {
     run : { module: serverModule, transport: TransportKind.ipc },
-    debug: { module: serverModule, transport: TransportKind.ipc /* *, options: debugOptions /* */ }
```

Comment:
Please add the comments back for commit to master. They should only be removed during testing. Also note that simply adding a single '/' after the second start has the same effect as deleting the comments. That explains the funny formatting of the comment (http://removingalldoubt.com/programming/2005/09/03/Comment-Toggle)